src/QMCDrivers/WFOpt/OutputMatrix.cpp

```diff
@@ -0,0 +1,60 @@
+//////////////////////////////////////////////////////////////////////////////////////
+// This file is distributed under the University of Illinois/NCSA Open Source License.
+// See LICENSE file in top directory for details.
+//
+// Copyright (c) 2021 QMCPACK developers.
+//
+// File developed by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//
+// File created by: Mark Dewing, mdewing@anl.gov, Argonne National Laboratory
+//////////////////////////////////////////////////////////////////////////////////////
+
+
+/** @file OutputMatrix.cpp
+ * @brief Output the Hamiltonian and overlap matrices from linear method
+ */
+
+#include "QMCDrivers/WFOpt/OutputMatrix.h"
```

Comment:
OutputMatrix.cpp needs to include "OutputMatrix.h". Related header files should be included without any path.